CHANGELOG.md

```diff
@@ -1,5 +1,38 @@
 # Changelog
 
+## v6.0.0 - 2018-01-09
+
+### Breaking changes
+
+- Remove `sc` alias which conflicts with `sc.exe` (#5827)
+- Separate group policy settings and enable policy controlled logging in PowerShell Core (#5791)
+
+### Engine updates and fixes
+
+- Handle `DLLImport` failure of `libpsrpclient` in PowerShell Remoting on Unix platforms (#5622)
+
+### Test
+
+- Replace `lee.io` Tests with `WebListener` (#5709) (Thanks @markekraus!)
+- Update the docker based release package tests (#5692)
```

Comment:
Maybe `Update the docker base release package tests due to Pester module removal and other issues`